Emby.Server.Implementations/LiveTv/LiveTvManager.cs

```diff
@@ -184,14 +182,6 @@ public Task<List<TunerHostInfo>> DiscoverTuners(bool newDevicesOnly, Cancellatio
             return EmbyTV.EmbyTV.Current.DiscoverTuners(newDevicesOnly, cancellationToken);
         }
 
-        void service_DataSourceChanged(object sender, EventArgs e)
-        {
-            if (!_isDisposed)
-            {
-                _taskManager.CancelIfRunningAndQueue<RefreshChannelsScheduledTask>();
```

Comment:
why is this removed?

Reply:
Because it isn't being called anymore: https://github.com/jellyfin/jellyfin/pull/797/files#diff-37059782617556caa1d6d7b72614670fL130

Reply:
I saw that you removed the event... but what's the reason for removal? and wasn't this task needed for some reason?